.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: Run Tests
+on: [push, pull_request]
+
+jobs:
+  tests:
+    name: Run Tests
+    defaults:
+      run:
+        shell: bash -l {0}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Do we want to start testing against python 3.9?

Reply:
That's a good point, we probably should. I'll add that.